CHANGELOG.md

```diff
@@ -34,8 +36,6 @@ The most recent changes are on top, in each type of changes category.
 
 - Improve scene time filtering performance [#610](https://github.com/cyfronet-fid/sat4envi/issues/610)
 
-### Updated
```

Comment:
Why have you remove it?

Reply:
I didn't remove it during release as I would normally do, so I do it here